specification/sql/resource-manager/Microsoft.Sql/preview/2018-06-01-preview/managedDatabases.json

```diff
@@ -638,6 +639,13 @@
           "x-ms-mutability": [
             "create"
           ]
+        },
```

Comment:
Is there a reason that 2019-06-01-preview version does not have this and LongTermRetentionManagedInstanceBackups.json?

Reply:
Added managedDatabases.json to 2019-06-01-preview as well.  the Backups/Policies APIs are new and we are going to use 2018-06-01-preview for those.